### ISSUE TITLE
Access scrcpy connected phone from another android phone?

### ISSUE THREAD

**Issue by bladyle** [Genymobile/scrcpy#3420]
I want to connect to a phone that is connected to my local network wirelessly. Is there a way or app for that?

**Comment by twaik** [comment]
https://gitlab.com/las2mile/scrcpy-android

**Comment by bladyle** [comment]
Yes I have found it but as you can see it's not maintained and it buggy.
